File: ios/MobilePassSDK/Model/DeviceCapability.swift
//
//  DeviceCapability.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 28.02.2021.
//

import Foundation

public struct DeviceCapability: Codable {
    var support:        Bool
    var enabled:        Bool
    var needAuthorize:  Bool
}

File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/DeviceCapability.java
package com.armongate.mobilepasssdk.model;

public class DeviceCapability {

    public boolean support;
    public boolean enabled;
    public boolean needAuthorize;

    public DeviceCapability(boolean support, boolean enabled, boolean needAuthorize) {
        this.support        = support;
        this.enabled        = enabled;
        this.needAuthorize  = needAuthorize;
    }

}
